<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogSet</class>
 <widget class="QDialog" name="LogSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="login_b">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_b">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_l">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pass_t">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
